src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1 class="main-title">Leaderboard</h1>
+    <main class="main">
+      <section class="section-one">
+         <h2 class="title">Recent scores <button type="button" class="refresh" id="refresh">Refresh</button></h2>
+         <ul class="score" id="scores">
+             <li class="score-name">Name: 100</li>
+             <li class="score-name one">Name: 20</li>
+             <li class="score-name">Name: 50</li>
+             <li class="score-name one">Name: 78</li>
+             <li class="score-name">Name: 125</li>
+             <li class="score-name one">Name: 77</li>
+             <li class="score-name">Name: 42</li>
+         </ul>
+      </section>
+      <section class="section-two">
+         <h2 class="title-form">Add your score</h2>
+         <form action="#" class="form">
+            <input type="text" id="name" class="name" placeholder="Your name">
+            <input type="text" id="scoreNumber" class="scor" placeholder="Your score">
+            <button type="submit" class="submit">Submit</button>
+         </form>
```

Comment:
- [ ] [OPTIONAL] Hi @Azonkeu, I congratulate you upon setting up your project, It really looks great so far! kudos 💪🏻. However I have a few optional suggestions on the form fields and think it would be nice if you added some left and right padding to the inputs and button because it would allow breathing space between the user input and the input itself 😄.

Reply:
Thank you for your suggestion. I will do that right away.